File: iteration2/src/CSE3063F22P1_GRP9/TakenCourse.java
package CSE3063F22P1_GRP9;

//this class used for course status (pass & fail & current) and grade informations.

public class TakenCourse {
	private Course course;
	private String takenCourseStatus;
	private float grade;
	
	public TakenCourse(Course course,float grade,String takenCourseStatus){
		this.course = course;
		this.grade = grade;
		this.takenCourseStatus = takenCourseStatus;
	}

	public Course getCourse() {
		return course;
	}

	public String getTakenCourseStatus() {
		return takenCourseStatus;
	}

	public void setTakenCourseStatus(String takenCourseStatus) {
		this.takenCourseStatus = takenCourseStatus;
	}

	public float getGrade() {
		return grade;
	}

	public void setGrade(float grade) {
		this.grade = grade;
	}
	
}

File: iteration3/src/TakenCourse.py
class TakenCourse:

    def __init__(self,course,grade,takenCourseStatus):
        self._course = course
        self._grade = grade
        self._takenCourseStatus = takenCourseStatus
    
    def getCourse(self):
        return self._course
    
    def getTakenCourseStatus(self):
        return self._takenCourseStatus
    
    def setTakenCourseStatus(self,takenCourseStatus):
        self._takenCourseStatus = takenCourseStatus
    
    def getGrade(self):
        return self._grade
    
    def setGrade(self,grade):
        self._grade = grade